src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/constants/additionalFilters.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+export const ADDITIONAL_FILTERS = {
+    programStage: 'programStage',
+    occurredAt: 'eventOccurredAt',
```

Comment:
Hey @simonadomnisoru, I noticed that the `report date` is not showing up in the list, this is due to unmatched term here, I believe changing this to `occurredAt` and add the field to the query should fix the problem
<img width="1400" alt="Screenshot 2022-10-19 at 09 31 25" src="https://user-images.githubusercontent.com/89806888/196639212-92b0ffb2-c1ac-4e36-aba0-d0adbc37c813.png">

Reply:
Hey @jasminenguyennn 

The `occurredAt` id is already used by the TEI filters. The list of filters doesn't work with duplicate filter ids. I need to use a different id for the program stage filters and that is why I choose to use `eventOccurredAt`. 

There is a converter to the API form when [filtering by report date](https://github.com/dhis2/capture-app/pull/3062/files#diff-f5af0891c6740a4d8327fd8aadb1c63a8fc5e99b84fb715312835f2bf180a38dR169). In my last commit, I also added a converter for ordering by report date and one for the report date values in the list. 

Let me know if it works fine now or if you find any other issues. Thank you!